server/db/db.js

```diff
@@ -0,0 +1,179 @@
+        table.unique(['script_id', 'question_id']);
+      });
+    }
+    return exist;
+  });
+
+const createCampaignsTable = () =>
+  bookshelf.knex.schema.hasTable('campaigns').then((exist) => {
+    if (!exist) {
+      return bookshelf.knex.schema.createTable('campaigns', (table) => {
+        table.increments('id').primary();
+        table.string('name').notNullable().unique().index();
+        table.string('title').notNullable();
+        table.string('description').notNullable();
+        table.enu('status', ['draft', 'active', 'pause', 'completed']).notNullable();
+        table.integer('contact_lists_id').references('contact_lists.id').notNullable();
+        table.integer('script_id').references('scripts.id').notNullable();
+        table.timestamp('created_at').defaultTo(bookshelf.knex.fn.now());
+        table.timestamp('updated_at').defaultTo(bookshelf.knex.fn.now());
+      });
+    }
+    return exist;
+  });
+
+const createUsersTable = () =>
+  bookshelf.knex.schema.hasTable('users').then((exist) => {
+    if (!exist) {
+      return bookshelf.knex.schema.createTable('users', (table) => {
+        table.increments('id').primary();
+        table.string('email').notNullable().unique().index();
+        table.string('first_name').notNullable();
+        table.string('last_name').notNullable();
+        table.string('phone_number').notNullable();
+        table.string('password_hash').notNullable();
+        table.boolean('is_admin').defaultTo(false);
+        table.boolean('is_banned').defaultTo(false);
+        table.boolean('is_active').defaultTo(true);
+        table.timestamp('created_at').defaultTo(bookshelf.knex.fn.now());
+        table.timestamp('updated_at').defaultTo(bookshelf.knex.fn.now());
+      });
+    }
+    return exist;
+  });
+
+const setup = () =>
+  createContactsTable()
+  .then(createContactListsTable)
+  .then(createContactsContactListsTable)
+  .then(createScriptsTable)
+  .then(createQuestionsTable)
+  .then(createQuestionsScriptsTable)
+  .then(createCampaignsTable)
+  .then(createUsersTable)
+  .then(() => {
+    console.log('Created all tables');
+  })
+  .catch((err) => {
+    console.error('Error creating tables: ', err);
+  });
+
+const teardown = () =>
+  bookshelf.knex.schema.dropTable('campaigns')
+  .then(() => bookshelf.knex.schema.dropTable('questions_scripts'))
+  .then(() => bookshelf.knex.schema.dropTable('questions'))
+  .then(() => bookshelf.knex.schema.dropTable('scripts'))
+  .then(() => bookshelf.knex.schema.dropTable('contacts_contact_lists'))
+  .then(() => bookshelf.knex.schema.dropTable('contact_lists'))
+  .then(() => bookshelf.knex.schema.dropTable('contacts'))
+  .then(() => bookshelf.knex.schema.dropTable('users'))
+  .then(() => {
+    console.log('Dropped all tables');
+  })
+  .catch((err) => {
+    console.error('Error dropping tables', err);
+  });
+
+export default bookshelf;
+export { bookshelf, setup, teardown };
```

Comment:
I export the object as default for all the models to pull in and then the helper scripts for setup and teardown need access to the named methods as well.